scripts/tag-custom-build.sh

```diff
@@ -40,10 +40,20 @@
 
 set -euo pipefail
 
+use_jj=false
+if [[ "${1-}" == "--jj" ]] ; then
+    use_jj=true
+    shift
+fi
+
 SHA="${1-}"
 
 if [ -z "$SHA" ] ; then
-    SHA="$(git rev-parse HEAD)"
+    if [ "$use_jj" = true ] ; then
```

Comment:
Nit: having the variable be empty initially then set it to `1` (or any string) if `--jj` is passed would allow you to use `-z` instead of `= true` which I think is a little more foolproof (you could misspell the string `true`, etc.)

No big deal if you keep it the way it is.